docs-website/sidebars.js

```diff
@@ -294,45 +293,6 @@ module.exports = {
           id: "docs/api/openapi/openapi-usage-guide",
         },
         "docs/dev-guides/timeline",
-        {
-          "Rest.li API": [
```

Comment:
we can't quite delete all the rest.li stuff until the functionality has been ported over to openapi